docs/en/observability/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -330,6 +325,7 @@ NOTE: Starting from **v1.4.0**, if you want to update the Events settings for th
       ElasticServerlessForwarderS3ConfigFile = ""
       ElasticServerlessForwarderSSMSecrets = ""
       ElasticServerlessForwarderKMSKeys = ""
+      ElasticServerlessForwarderAssumeRoles = ""
```

Comment:
this must be removed: it is not needed.
probably a leftover since you started the review and I pushed the latest change